app/javascript/react/components/BayComponent.js

```diff
@@ -1,11 +1,11 @@
 import React, { useState, useEffect } from "react"
 
 const BayComponent = (props) => {
-  return(console.log("Page test."))
+  const brandId = (props.match.params.id)
+  return(
+    <h1>{`${brandId}`}</h1>
+  )
```

Comment:
in addition, while you are successful in linking to this page, it doesnt really do much. This would normally be rejected. What is the purpose of line 6